com.unity.postprocessing/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [3.1.3] -
```

Comment:
We haven't publish the 3.1.2 version, this is where the changelog should go. It is a bit misleading like this but it is requried to have this format: ## [3.1.2] - 2021-02-06 to pass Yamato test. When we produce a new version we bump it

Reply:
I moved the changelog to 3.1.2